config/default.yml

```
@@ -153,6 +153,8 @@ AllCops:
     rubocop-sequel: [sequel]
     rubocop-rake: [rake]
     rubocop-graphql: [graphql]
+  # Enable/Disable checking the methods extended by ActiveSupport.
```

That's my two cents.

```suggestion
  # Enable/Disable checking the methods extended by Active Support.
```

> The proper names of Rails components have a space in between the words, like "Active Support". `ActiveRecord` is a Ruby module, whereas Active Record is an ORM. 

https://guides.rubyonrails.org/api_documentation_guidelines.html#wording

---

Thank you. I've corrected it along with other location.